draft-ietf-rtcweb-jsep.xml

```diff
@@ -993,7 +993,30 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                     the sessions that it no longer needs.</t>
 
                 </section>
-            </section>
+              </section>
+
+              <section  title="Simulcast" anchor="sec.simulcast">
+
+		        <t> JSEP supports simulcast using multiple encodings of the
+		        source media within the same m-line section of the SDP with
+		        additional SDP marking to allow clients that support simulcast
+		        to understand that these are simulcasts of the same source
+		        media. Clients that don't support simulcast will end up
+		        selecting only of the encodings and functionality in a non
+		        simulcast way. </t>
+	      
+                <t> When using JSEP to send or receive multiple encodings of the
+                same media source, the techniques from <xref
+                target="I-D.ietf-mmusic-sdp-simulcast"/> and <xref
+                target="I-D.ietf-mmusic-rid" /> are used. An offerer wanting to
+                use simulcast MUST include the "a=simulcast" attribute in the
```

Comment:
is this "wanting to send", "wanting to receive", "both", or is that controlled by further parameters?